common/src/main/java/com/microsoft/identity/common/internal/commands/DeviceCodeFlowCommand.java

```diff
@@ -38,6 +40,16 @@
  * exception handling.
  */
 public class DeviceCodeFlowCommand extends TokenCommand {
+
+    public final static String AUTH_DECLINED_CODE = "authorization_declined";
+    public final static String AUTH_DECLINED_MSG = "The end user has denied the authorization request. Re-run the Device Code Flow Protocol with another user.";
+    public final static String EXPIRED_TOKEN_CODE = "expired_token";
+    public final static String EXPIRED_TOKEN_MSG = "The token has expired, therefore authentication is no longer possible with this flow attempt. Re-run the Device Code Flow Protocol to try again.";
+    // Add more errors
+
+    // Use this message for when DCF fails with an error code that doesn't match any of the codes above
+    public final static String DEFAULT_ERROR_MSG = "Device Code Flow has failed with an unexpected error. The error code shown was received from the authorization result.";
```

Comment:
I don't think these should live here. They should instead be in `ErrorStrings.java`

Reply:
Also following the same pattern in that class, let's use complete variable names

Reply:
+1 on this feedback, let's organize these constants into `ErrorStrings`

Reply:
Done!